src/functions.js

```diff
@@ -1,12 +1,43 @@
-const addScore = () => {
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+
+const addNewGame = (endpoint = '', gameName = '') => {
+  const targetUrl = url + endpoint;
+  const data = { name: gameName };
+  return fetch(targetUrl, {
+    method: 'POST',
+    headers: {
+      'content-type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  }).then((resp) => resp.json());
```

Comment:
- [ ] <b>REQUIRED:</b> As requested in project requirements, you have to use `async/await` in case of any API request in this project. Kindly do that to comply with the project specifications.